src/openrct2-ui/interface/FileBrowser.h

```diff
@@ -0,0 +1,63 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2025 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <cstdint>
+#include <functional>
+#include <openrct2/core/StringTypes.h>
+#include <openrct2/localisation/StringIdType.h>
+#include <string>
+
+struct TrackDesign;
+
+namespace OpenRCT2
+{
+    struct WindowBase;
+}
+
+namespace OpenRCT2::Ui::FileBrowser
+{
+    enum class FileType : uint8_t
+    {
+        directory,
+        file,
+    };
+
+    struct LoadSaveListItem
+    {
+        std::string name{};
+        std::string path{};
+        time_t date_modified{ 0 };
```

Comment:
I suppose updating these names is outside the scope?

Reply:
It's not that big a change, so might as well tackle this now.